src/AzureMcp.csproj

```diff
@@ -18,6 +18,7 @@
     <PublishTrimmed>false</PublishTrimmed>
     <IncludeNativeLibrariesForSelfExtract>true</IncludeNativeLibrariesForSelfExtract>
     <DebugType>embedded</DebugType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Why is this required?

Reply:
This is required by the code to get the WindowHandle.